packages/widget-chat/src/components/wrapper-scrollable/index.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+
+import styles from './styles.css';
+
+function getDisplayName(C) {
+  return C.displayName || C.name || `C`;
+}
+
+export default function scrollableComponent(WrappedComponent) {
+  class InjectScrollable extends Component {
+    constructor(props) {
+      super(props);
+      this.getNode = this.getNode.bind(this);
```

Comment:
yep, that's the one. i'm pretty much always for adding dependencies over building it ourselves.